app/views/dishes/show.html.haml

```diff
@@ -14,4 +14,5 @@ Menu(s):
 - @dish.menus.each do |menu|
   = menu.title
 %br/
-= link_to 'Add to cart', dish_add_item_path(@dish), method: :post, class: "btn btn-success"
+- unless !current_user.nil? && current_user.role == 'owner'
```

Comment:
Nah cuz visitors can add things to their cart. Will have to make it `current_user.owner? || current_user.admin?` when we get around to adding admins, but everybody else should see that button